README.rst

```diff
@@ -4,109 +4,114 @@ NVIDIA DALI
 ===========
 .. overview-begin-marker-do-not-remove
 
-Deep learning applications require complex, multi-stage pre-processing
-data pipelines. Such data pipelines involve compute-intensive operations
-that are carried out on the CPU. For example, tasks such as: load data
-from disk, decode, crop, random resize, color and spatial augmentations
-and format conversions, are mainly carried out on the CPUs, limiting the
-performance and scalability of training and inference.
-
-In addition, the deep learning frameworks have multiple data
-pre-processing implementations, resulting in challenges such as
-portability of training and inference workflows, and code
-maintainability.
-
-NVIDIA DALI (R), NVIDIA's Data Loading Library, is a collection of highly
-optimized building blocks, and an execution engine, to accelerate the
-pre-processing of the input data for deep learning applications. DALI
-provides both the performance and the flexibility to accelerate
-different data pipelines as one library. This library can
-then be easily integrated into different deep learning training and
-inference applications.
+The NVIDIA Data Loading Library (DALI) is a library for data loading and
+pre-processing to accelerate deep learning applications. It provides a
+collection of highly optimized building blocks for loading and processing
+image, video and audio data, and it can be used as a portable drop-in replacement
+for built in data loaders and data iterations in popular deep learning frameworks.
+
+Deep learning applications require complex, multi-stage data processing pipelines
+that include loading, decoding, cropping, resizing, and many other augmentations.
+These data processing pipelines, which are currently executed on the CPU, have become a 
+bottleneck, limiting the performance and scalability of training and inference.
+
+DALI addresses the problem of the CPU bottleneck by offloading data preprocessing to the
+GPU. Additionally, DALI relies on its own execution engine, built to maximize the throughput
+of the input pipeline. Features such as prefetching, parallel execution, and batch processing
+are handled transparently for the user.
+
+In addition, the deep learning frameworks have multiple data pre-processing implementations,
+resulting in challenges such as portability of training and inference workflows, and code
+maintainability. Data processing pipelines implemented using DALI are portable because they
+can easily be retargeted to TensorFlow, PyTorch, MXNet and PaddlePaddle. 
+
+.. image:: /dali.png
```

Comment:
It seems that this file `dali.png` is empty in this PR.

Reply:
Only in the diff view. If you click "View file" you can see the file